conda/environments/all_cuda-115_arch-x86_64.yaml

```diff
@@ -0,0 +1,48 @@
+# This file is generated by `rapids-dependency-file-generator`.
+# To make changes, edit ../../dependencies.yaml and run `rapids-dependency-file-generator`.
+channels:
+- rapidsai
+- rapidsai-nightly
+- conda-forge
+- nvidia
+dependencies:
+- GPUtil>=1.4.0
+- c-compiler
+- click
+- cmake>=3.23.1,!=3.25.0
+- cudatoolkit=11.5
+- cupy >=10,<12.0.0a0
+- cxx-compiler
+- gcc_linux-64=9.*
+- imagecodecs>=2021.6.8
+- ipython
+- jbig
+- jpeg
+- libwebp-base
+- nbsphinx
+- ninja
+- numpy
+- numpydoc
+- nvcc_linux-64=11.5
+- openslide-python>=1.1.2
+- pip
+- pre-commit
+- psutil>=5.8.0
+- pydata-sphinx-theme
+- pytest-cov>=2.12.1
+- pytest-lazy-fixture>=0.6.3
+- pytest-xdist
+- pytest>=6.2.4
+- python>=3.8,<3.10
+- recommonmark
+- scikit-image >=0.19.0,<0.20.0a0
+- scipy
+- sphinx
+- sysroot_linux-64==2.17
+- tifffile>=2022.7.28
+- xz
+- zlib
```

Comment:
It looks like `yasm` is missing from here (at least for `x86_64`). Should we be handling that some other way?

Reply:
Replied here: https://github.com/rapidsai/cucim/pull/471/files#r1063810045

I didn't know what section of the dependencies to put it in.